specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/alerts.json

```diff
@@ -556,12 +556,596 @@
         "type": "object"
       },
       "description": "Changing set of properties depending on the entity type.",
+      "discriminator": "type",
       "properties": {
         "type": {
           "readOnly": true,
           "type": "string",
           "description": "Type of entity"
         }
+      },
+      "required": [
+        "type"
+      ]
+    },
+    "AccountEntity": {
```

Comment:
The PR seems adding a lot of types, but I didn't see the place using them (no new API, no new property in existing type, just new types).

Reply:
They are all inherit from #/definitions/AlertEntity.
They are specific implementations of that property, we need this change for 2 reasons:
1) So SDK users will have an easier time using the specific type
2) So there will be documentation of passible return values